zerver/webhooks/freshdesk/fixtures/inline_images.json

```diff
@@ -9,6 +9,6 @@
         "ticket_status":"Open",
         "ticket_priority":"Urgent",
     "requester_name":"Requester ☃ Bob",
-        "requester_email":"requester-bob@example.com",
+        "requester_email":"requester-bob@example.com"
```

Comment:
Freshdesk isn’t really going to send us invalid JSON…right?

Reply:
Ugh, this one is our fault, we provide an invalid custom “JSON” format in our [documentation](https://zulip.com/integrations/doc/freshdesk).

Reply:
Hmm, annoying.  We can email the relevant organization administrators.